src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -27,11 +27,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        ' Protected MustOverride Function StatementKind() As OperationKind
-
         Protected Overridable Function StatementKind() As OperationKind
```

Comment:
It is intended that the StatementKind method actually become MustOverride. The comment was an aspirational reminder.